mountpoint-s3-fs/src/config.rs

```diff
@@ -38,6 +38,7 @@ impl MountpointConfig {
         s3_path: S3Path,
         client: Client,
         runtime: Runtime,
+        error_callback: Option<ErrorCallback>,
```

Comment:
Couldn't this be a field in `MountpointConfig`?

Reply:
The field `S3Filesystem::config` is private, so either:

1) make the field public
2) add a getter to `S3Filesystem` for `config` field (or for `error_callback`)
3) store it in `S3FuseFilesystem` directly (as implemented now)

What do you think will be more appropriate?